packages/gafl-webapp-service/src/plugins.js

```diff
@@ -0,0 +1,93 @@
+import Inert from '@hapi/inert'
+import Vision from '@hapi/vision'
+import Disinfect from 'disinfect'
+import Scooter from '@hapi/scooter'
+import Blankie from 'blankie'
+import Crumb from '@hapi/crumb'
+import HapiGapi from '@defra/hapi-gapi'
+import { useSessionCookie } from './session-cache/session-manager.js'
+import { UTM } from './constants.js'
+
+// This is a hash of the inline script at line 31 of the GDS template. It is added to the CSP to except the in-line
+// script. It needs the quotes.
+const scriptHash = "'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU='"
+
+export const getPlugIns = (getSessionCookieName, getCsrfTokenCookieName) => {
```

Comment:
getSessionCookieName & getCsrfTokenCookieName should be exported elsewhere and added as an import here rather than passing references.

Plugin is normally treated as one word so method should be getPlugins()

Reply:
That is a matter of opinion not fact and as such I will not do it as an independent contractor I do the job according to my judgement; you really need to reduce the scope of your comments to those which have a material importance.

Reply:
For a quiet lift I have refactored this, but I think you should confine your comment to that which is materially significant. In the end this is a matter of taste - as the plugin file is really just an extension of the server file rather than being a separate logical module it is perfectly reasonable to pass a function reference which is primarily used in the main module file - in fact it serves to illustrate the relationship between the files. As an independent contractor I reserve the right to make these sort of decisions as to my own judgement.

Reply:
Isn't the purpose of a review to seek the opinion of others?